src/routes/tekenmethodes/[slug]/voorbeelden/+page.svelte

```diff
@@ -72,7 +71,12 @@
       {#each data.methods as method}
         {#if method.examples.length > 0}
           {#each method.examples as example}
-            <img class="carrousel-img" src={example.url} alt="Example Slide" />
+            <img
+              class="carrousel-img"
+              src={example.url}
+              alt="Example Slide"
+              loading="lazy"
```

Comment:
Zorg ervoor dat je lazy loading niet toevoegd aan foto's die meteen in beeld zijn.